rslearn/data_sources/local_files.py

```diff
@@ -353,14 +360,16 @@ def ingest_item(
                 aux_files.append(path.parent / (prefix + ext))
 
         # TODO: move converting fiona file to list[Feature] to utility function.
-        # TODO: don't assume WGS-84 projection here.
         with get_upath_local(path, extra_paths=aux_files) as local_fname:
             with fiona.open(local_fname) as src:
+                crs = CRS.from_wkt(src.crs.to_wkt())
+                projection = Projection(crs, 1, 1)
```

Comment:
Suggestion: Do we want the resolution to always be hardcoded as 1? If yes, prefer using the keywords here for readability. If not should be configurable.